junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/Arguments.java

```diff
@@ -12,7 +12,12 @@
 
 import static org.apiguardian.api.API.Status.EXPERIMENTAL;
 
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
```

Comment:
Don't import types that are only used in Javadoc comments. Use FQCN in the comments instead.

Reply:
I'm already rephrasing it a bit and can take care of that, too.